src/providers/coinMarketCap.ts

```diff
@@ -108,24 +123,22 @@ const HISTORICAL_OPTIONS = {
   json: true
 }
 
-const ascoinMarketCapHistoricalQuotes = asMap(
-  asObject({
-    symbol: asString,
-    quotes: asArray(
-      asObject({
-        timestamp: asString,
-        quote: asMap(
-          asObject({
-            price: asNumber
-          })
-        )
-      })
-    )
-  })
-)
+const ascoinMarketCapHistoricalQuotes = asObject({
```

Comment:
Since you removed the asMap, the code that uses this cleaner needs to change but it didn't. How does this possibly work?

See its use as a type in `coinMarketCapHistoricalQuote`